python/pyarrow/_flight.pyx

```diff
@@ -232,6 +237,34 @@ cdef class FlightDescriptor:
                 type(descriptor)))
         return (<FlightDescriptor> descriptor).descriptor
 
+    def serialize(self):
+        """Get the wire-format representation of this type.
+
+        Useful when interoperating with non-Flight systems (e.g. REST
+        services) that may want to return Flight types.
+
+        """
+        cdef c_string out
+        check_status(self.descriptor.SerializeToString(&out))
+        return out
+
+    @classmethod
+    def parse(self, serialized):
```

Comment:
Call this `unserialize`?

Reply:
or deserialize

Reply:
I made it `serialize`/`deserialize` across all languages in the end.